src/FieldDescription/TypeGuesserInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\FieldDescription;
+
+use Sonata\AdminBundle\Admin\FieldDescriptionInterface;
+use Symfony\Component\Form\Guess\TypeGuess;
+
+interface TypeGuesserInterface
+{
+    public function guess(FieldDescriptionInterface $fieldDescription): TypeGuess;
```

Comment:
Should we create a TypeGuesserChain too ?

Reply:
I wasn't sure, I guess so, I'll take a look.